src/database-controller/sdk/index.js

```diff
@@ -0,0 +1,301 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+const { Sequelize, Model } = require('sequelize');
+
```

Comment:
Have you tested against other SQL db?

Reply:
I didn't test. But sequelize supports it according to its document.

Reply:
Pls test it or at least track an issue for it

Reply:
Issued. see https://github.com/microsoft/pai/issues/4793